frontend-zs/src/components/stories/StoryList.js

```diff
@@ -14,7 +14,7 @@ export default function StoryList() {
     const [displayCategories, setDisplayCategories] = useState([])
 
     useEffect(() => {
-        Axios.get(process.env.REACT_APP_HOST + ':' + process.env.REACT_APP_PORT + '/api/stories')
+        Axios.get(process.env.REACT_APP_API_URL + ':' + process.env.REACT_APP_API_PORT + '/api/stories')
```

Comment:
After discussing we agreed that we will stick with `REACT_APP_PORT  `and `REACT_APP_HOST` as env variables. Sorry we didn’t manage the `.env` file very well